.github/scripts/version_manager.sh

```
@@ -412,18 +521,62 @@ main() {
+                echo "📊 버전 불일치 감지: version.yml($yml_version) vs build.gradle($gradle_version)"
+                # 두 버전 중 높은 버전 선택
+                local higher_version=$(get_higher_version "$yml_version" "$gradle_version")
+                
+                # 높은 버전으로 양쪽 모두 동기화
+                echo_info "버전 동기화: $higher_version"
+                # version.yml 업데이트
+                update_version_yml "$higher_version"
+                CURRENT_VERSION="$higher_version"
+                
+                # build.gradle 업데이트 (루트 및 모든 하위 모듈)
+                for f in $(find . -maxdepth 2 -name build.gradle -type f); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)